site/app/models/Gradeable.php

```diff
@@ -331,6 +320,8 @@ public function __construct(Core $core, $details, User $user = null) {
                         if ($details['array_gcd_gc_id'][$j] === $component_details['gc_id']) {
                             $component_details['gcd_score'] = $details['array_gcd_score'][$j];
                             $component_details['gcd_component_comment'] = $details['array_gcd_component_comment'][$j];
+                            $component_details['gcd_grader_id'] = $details['array_gcd_grader_id'][$j];
```

Comment:
They do not. All of the grader User details are set in [these fields](https://github.com/Submitty/Submitty/blob/ff2222158362abcdd6d15087279dd4a84462307a/site/app/models/Gradeable.php#L300-L301) (prefixed by `array_gcd_`) coming from the query.